### PR TITLE
Artifact Registry migration

### PR DESCRIPTION
**Pull request by takumiohym**
Migration from Container Registry to Artifact Registry due to the product deprecation.

### CODE REVIEW

File: notebooks/docker_and_kubernetes/solutions/1_intro_docker.ipynb [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "\n",
```
Comment (takumiohym) [inline notebooks/docker_and_kubernetes/solutions/1_intro_docker.ipynb:8]: <p>Added this authorization command where <code>docker</code> command is needed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

**Review by sanjanalreddy — APPROVED** [review]
LGTM. Thank you for modifying all the files, Takumi!

**Review by BenoitDherin — APPROVED** [review]
LGMT

**Comment by takumiohym** [comment]
Thank you!

**Pull request merged**